### PR TITLE
Adding identity_uri to /lightning/get-info

### PR DESCRIPTION
**Pull request by rootzoll**
Open naming question: https://github.com/fusion44/blitz_api/issues/88#issuecomment-1128010617

Also the PR will just deliver the node URI on raspiblitz platform for now.

### CODE REVIEW

**Comment by rootzoll** [comment]
OK moved to "/lightning/get-info" ... only works for RASPIBLITZ Platform for now

**Comment by rootzoll** [comment]
@fusion44 ready to merge

File: app/models/lightning.py [hunk 1]
```diff
@@ -950,7 +950,10 @@ class LnInfo(BaseModel):
     commit_hash: str
 
     # The identity pubkey of the current node.
-    identity_pubkey: str
+    identity_pubkey: str = Query("the nodes pubkey")
+
+    # The complete URI (pubkey@physicaladdress:port) the current node.
+    identity_uri: str = Query("the nodes comnplete URI")
```
Comment (fusion44) [inline app/models/lightning.py:957]: "comnplete" :P If you are using vscode I can recommend the "Code Spell Checker" extension.

File: app/models/lightning.py [hunk 2]
```diff
@@ -950,7 +950,10 @@ class LnInfo(BaseModel):
     commit_hash: str
 
     # The identity pubkey of the current node.
-    identity_pubkey: str
+    identity_pubkey: str = Query("the nodes pubkey")
```
Comment (fusion44) [inline app/models/lightning.py:953]: If you use a Query a default value is recommended. Either and empty string or just None if optional.
```python
Query("" ,"The nodes lightning pubkey. Currently an empty string on platforms other than RaspiBlitz.")
```

**Review by fusion44 — CHANGES_REQUESTED** [review]
**pedantic mode on** Please fix this typo, otherwise it's all good. **pedantic mode off**

File: app/models/lightning.py (hunk 1 above)
Comment (rootzoll) [reply app/models/lightning.py:957]: comitted

File: app/models/lightning.py (hunk 2 above)
Comment (rootzoll) [reply app/models/lightning.py]: comitted

**Comment by rootzoll** [comment]
``
identity_pubkey: str = Query("","the nodes pubkey")
May 17 22:57:16 python[4160120]: TypeError: Query() takes 1 positional argument but 2 were given
``
reverting back to just one argument for now

**Comment by cstenglein** [comment]
@rootzoll does it get send with the `ln_info_lite` SSE event as well?

I'm currently not calling an API for connection info, I get everything over SSE :)

**Pull request merged**